community/server/src/main/java/org/neo4j/server/database/LifecycleManagingDatabase.java

```diff
@@ -114,4 +118,18 @@ public boolean isRunning()
     {
         return isRunning;
     }
+
+    private void preLoadCypherCompiler()
+    {
+        // Execute a single Cypher query to pre-load the compiler to make the first user-query snappy
+        //noinspection EmptyTryBlock
+        try (Result ignored = this.graph.execute("MATCH (a:` WEIRD LABEL NO-ONE SHOULD USE. EVER `) RETURN a"))
```

Comment:
Why not put in a LIMIT 1? Then a less weird label name can be used. I guess this will still execute a micro-transaction to create the label name if it does not already exist, right?